### DISCUSSION TITLE
sync_mode is set to Sync, but the logs show it's using Append

### DISCUSSION THREAD

**Discussion by fmr-iv** [meisnate12/Plex-Meta-Manager#412] [Q&A] [answered]
Here is my collections config:
```
collections:
  Trending Movies - Week:
    trakt_watched: 30
    trakt_recommended: 30
    trakt_trending: 30
    build_collection: false
    item_label: Trending Movies - Week
    sync_mode: sync
```

And here is a picture of the run (note the "**Sync Mode: append**" at the top):
![image](https://user-images.githubusercontent.com/4317891/138325780-40b847d2-7b3c-41ed-a671-0dd0eedde23f.png)

Any help would be appreciated!

**Comment by meisnate12** [comment]
Can you post your meta.log file

**Reply by fmr-iv** [reply]
[meta.log](https://github.com/meisnate12/Plex-Meta-Manager/files/7391702/meta.log)
[collection.log](https://github.com/meisnate12/Plex-Meta-Manager/files/7391708/collection.log)

**Comment by meisnate12** [comment, marked as answer]
ok so basically this isnt how this is intended to function. 

`build_collection` doesnt work with `sync_mode` because well theres no collection to sync with you basically want the labels your adding to sync. 

`item_label` doesnt have anything to do with sync.

what you want is a [`smart_label`](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Smart-Builders#smart-label) collection, its exactly what youre trying to do its just already built into PMM.

```yaml
collections:
  Trending Movies - Week:
    trakt_watched: 30
    trakt_recommended: 30
    trakt_trending: 30
    sync_mode: sync
    smart_label: random
```

the sort you give smart label will be how the collection is sorted all sort options are [here](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Smart-Builders#sort-options).

What this does is add a label to all the items that is the same as the mapping name for the collection and then creates a smart collection only looking at those labels.

**Reply by fmr-iv** [reply]
Thanks for the info! I'll give it a shot
